support_bundle/get_support_bundle.sh

```diff
@@ -1,50 +1,153 @@
-        done
-    fi
+    kubectl ${KUBE_OPTS} get pod ${pod} -o json > ${LOG_DIR}/${pod}/kubectl-describe.json
+    containers=$(kubectl ${KUBE_OPTS} get pod ${pod} -o json | jq -r '.spec.containers[].name')
+    for container in ${containers}; 
+    do
+        kubectl ${KUBE_OPTS} logs ${pod} -c ${container} > ${LOG_DIR}/${pod}/${container}-kubectl-logs.txt
+        kubectl ${KUBE_OPTS} exec ${pod} -c ${container} -- bash -c "${command}" > ${LOG_DIR}/${pod}/${container}-support-files.tgz || true
+    done
 done
 
-for object in svc deployment sts pvc daemonset ingress replicaset; do
+# Get info on deployments, statefulsets, persistentVolumeClaims, daemonsets, and ingresses
+for object in svc deployment sts pvc daemonset ingress replicaset; 
+do
     items=$(kubectl ${KUBE_OPTS} get ${object} -o jsonpath="{.items[*]['metadata.name']}")
     mkdir -p ${LOG_DIR}/${object}
-    for item in ${items}; do
+    for item in ${items}; 
+    do
         kubectl ${KUBE_OPTS} get ${object} ${item} -o json > ${LOG_DIR}/${object}/${item}-kubectl.json
     done
 done
 
-kubectl ${KUBE_OPTS} get configmap sysdigcloud-config -o yaml | grep -v password > ${LOG_DIR}/config.yaml || true
+# Fetch container density information
+num_nodes=0
+num_pods=0
+num_running_containers=0
+num_total_containers=0
+
+printf "%-30s %-10s %-10s %-10s %-10s\n" "Node" "Pods" "Running Containers" "Total Containers" >> ${LOG_DIR}/container_density.txt
+for node in $(kubectl ${KUBE_OPTS} get nodes --no-headers -o custom-columns=node:.metadata.name);
+do
+    total_pods=$(kubectl ${KUBE_OPTS} get pods -A --no-headers -o wide | grep ${node} |wc -l |xargs)
+    running_containers=$( kubectl ${KUBE_OPTS} get pods -A --no-headers -o wide |grep ${node} |awk '{print $3}' |cut -f 1 -d/ | awk '{ SUM += $1} END { print SUM }' |xargs)
+    total_containers=$( kubectl get ${KUBE_OPTS} pods -A --no-headers -o wide |grep ${node} |awk '{print $3}' |cut -f 2 -d/ | awk '{ SUM += $1} END { print SUM }' |xargs)
+    printf "%-30s %-15s %-20s %-10s\n" "${node}" "${total_pods}" "${running_containers}" "${total_containers}" >> ${LOG_DIR}/container_density.txt
+    num_nodes=$((num_nodes+1))
+    num_pods=$((num_pods+${total_pods}))
+    num_running_containers=$((num_running_containers+${running_containers}))
+    num_total_containers=$((num_total_containers+${total_containers}))
+done
+  
+printf "\nTotals\n-----\n" >> ${LOG_DIR}/container_density.txt
+printf "Nodes: ${num_nodes}\n" >> ${LOG_DIR}/container_density.txt
+printf "Pods: ${num_pods}\n" >> ${LOG_DIR}/container_density.txt
+printf "Running Containers: ${num_running_containers}\n" >> ${LOG_DIR}/container_density.txt
+printf "Containers: ${num_total_containers}\n" >> ${LOG_DIR}/container_density.txt
+
+# Fetch Cassandra Nodetool output
+echo "Fetching Cassandra statistics";
+mkdir -p ${LOG_DIR}/cassandra
+for pod in $(kubectl ${KUBE_OPTS} get pod -l role=cassandra | grep -v "NAME" | awk '{print $1}')
```

Comment:
I know this convention has been used throughout the script, but wouldn't it be cleaner just to use `kubectl .... --no-headers -o name` instead of using `grep` + `awk`?

and no need to remove the leading `pod/` in the name, it works as intended:
```bash
❯ kubectl get pod -l role=cassandra --no-headers -o name
pod/sysdigcloud-cassandra-0
pod/sysdigcloud-cassandra-1
pod/sysdigcloud-cassandra-2
❯ kubectl exec -ti pod/sysdigcloud-cassandra-0 -c cassandra -- nodetool info
ID                     : 9d176d27-af85-4d48-97dd-0091b76bf9c6
Gossip active          : true
Thrift active          : true
Native Transport active: true
[...]
```